src/util/parser.py

```diff
@@ -176,8 +176,8 @@ def add_argument_dry(argparser):
         "2. no_signer: Do not use signer",
         action="store",
         choices=[
-            DryRun.SIGNER,
-            DryRun.NO_SIGNER,
+            "SIGNER",
```

Comment:
I must admit that I hate Python more with every day.. you are telling me the following:
1) the default case casts a type Bool
2) the default is casting an enum class:string
3) the choices are just strings

I think this is terrible. At least we should do DryRun.SIGNER.value ? and consequently the action should properly store it as a value of the class? Doesn't this bother you?

Reply:
It absolutely bothered me.
Thank you for the suggestion of the .value